MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/annotations/ClickOnLayerActivity.java

```diff
@@ -0,0 +1,153 @@
+package com.mapbox.mapboxandroiddemo.examples.annotations;
```

Comment:
This seems more like a query map or runtime styling example rather than annotation.